src/main.py

```diff
@@ -107,6 +132,91 @@ def add_detectors(
         return message_list
 
 
+@dataclasses.dataclass
+class NextVisitModelKeda:
+    "Next Visit Message"
```

Comment:
[DM style guide requires triple quotes](https://developer.lsst.io/python/numpydoc.html#docstrings-should-begin-with-and-terminate-with-on-its-own-line) even for single-line docstrings.
```suggestion
    """Next Visit Message"""
```